seed/tests/test_compliance_metrics_views.py

```diff
@@ -52,7 +45,7 @@ def setUp(self):
         self.filter_group = FilterGroup.objects.create(
             name='filter group 1',
             organization_id=self.org.id,
-            inventory_type=1,  # Tax Lot
+            inventory_type=1,  # Property
```

Comment:
is there an ENUM for this? Is it [this](https://github.com/SEED-platform/seed/blob/compliance_metrics/seed/models/column_list_profiles.py#L25) because 1 is tax lot...

Reply:
@nllong actually I don't know what this is! I couldn't find the enum and I found some places where 1 was Property and 1 was Taxlot. I wanted to use property....

Reply:
nevermind, found it! property = 0, taxlot = 1.